### PR TITLE
feat(nav): add API schema links

### LINKED ISSUE #64: Add links to swagger file and schema editor
Currently there is no obvious way to get to BrokenCrystals' Swagger file or open it in the schema editor. We should add such links to the header navigation menu.

### PR DESCRIPTION
**Pull request by tamirGer**
closes #64

### CODE REVIEW

File: public/src/pages/main/Header/Nav.tsx [hunk 1]
```diff
@@ -64,6 +64,24 @@ export const Nav = () => {
...
+              <a href="https://brokencrystals.com/swagger/json" target="_blank">
+                OpenAPI 3.0 JSON
+              </a>
+            </li>
+            <li>
+              <a
+                href="https://brokencrystals.com/swagger/static/index.html"
+                target="_blank"
+              >
+                Schema Editor
```
Comment (derevnjuk) [inline public/src/pages/main/Header/Nav.tsx:80]:
```suggestion
                API Reference
```

**Review by derevnjuk — CHANGES_REQUESTED** [review]

**Review by derevnjuk — APPROVED** [review]

**Review by derevnjuk — COMMENTED** [review]
@tamirGer the main task implies refactoring API scheme itself. As we discussed, you should go through endpoints and adjust Swagger decorators to omit such declarations as Buffer, Readable, etc

See https://docs.nestjs.com/openapi/types-and-parameters#raw-definitions

**Comment by tamirGer** [comment]
@derevnjuk This PR is not related to #63, this is additional functionality requested by @ArtLinkov. This should just allow easier access to the SWAGGER.

**Comment by derevnjuk** [comment]
@tamirGer well, then you have to open a Github issue with some description and lint it to this PR

**Comment by tamirGer** [comment]
@derevnjuk done

**Pull request merged**